tests/src/test/scala/tests/commands/ExportCommandSuite.scala

```diff
@@ -800,4 +800,75 @@ class ExportCommandSuite extends tests.BaseSuite with tests.ConfigSyntax {
            |@maven//:org.apiguardian/apiguardian-api/apiguardian-api-1.1.1.jar""".stripMargin,
     )
   )
+
+  checkMultipleDeps(
+    "transitive dependencies and eviction",
+    deps(
+      dep("io.get-coursier:coursier-cli_2.12:2.0.0")
+        .target("coursier-cli"),
+      dep("io.get-coursier:coursier_2.12:2.0.13")
+        .target("coursier")
+    ),
+    queries = List(
+      allJars("@maven//:coursier-cli") ->
+        """|@maven//:com.chuusai/shapeless_2.12/shapeless_2.12-2.3.3.jar
+           |@maven//:com.github.alexarchambault/argonaut-shapeless_6.2_2.12/argonaut-shapeless_6.2_2.12-1.2.0.jar
+           |@maven//:com.github.alexarchambault/case-app-annotations_2.12/case-app-annotations_2.12-2.0.0.jar
+           |@maven//:com.github.alexarchambault/case-app-util_2.12/case-app-util_2.12-2.0.0.jar
+           |@maven//:com.github.alexarchambault/case-app_2.12/case-app_2.12-2.0.0.jar
+           |@maven//:com.github.mpilquist/simulacrum_2.12/simulacrum_2.12-0.10.0.jar
+           |@maven//:com.github.plokhotnyuk.jsoniter-scala/jsoniter-scala-core_2.12/jsoniter-scala-core_2.12-2.2.4.jar
+           |@maven//:com.google.collections/google-collections/google-collections-1.0.jar
+           |@maven//:com.squareup.okhttp3/okhttp/okhttp-3.13.1.jar
+           |@maven//:com.squareup.okio/okio/okio-1.17.2.jar
+           |@maven//:commons-io/commons-io/commons-io-2.6.jar
+           |@maven//:io.argonaut/argonaut_2.12/argonaut_2.12-6.2.5.jar
+           |@maven//:io.get-coursier/coursier-cache_2.12/coursier-cache_2.12-2.0.13.jar
+           |@maven//:io.get-coursier/coursier-cli_2.12/coursier-cli_2.12-2.0.0.jar
+           |@maven//:io.get-coursier/coursier-core_2.12/coursier-core_2.12-2.0.13.jar
+           |@maven//:io.get-coursier/coursier-env_2.12/coursier-env_2.12-2.0.0.jar
+           |@maven//:io.get-coursier/coursier-install_2.12/coursier-install_2.12-2.0.0.jar
+           |@maven//:io.get-coursier/coursier-jvm_2.12/coursier-jvm_2.12-2.0.0.jar
+           |@maven//:io.get-coursier/coursier-launcher_2.12/coursier-launcher_2.12-2.0.0.jar
+           |@maven//:io.get-coursier/coursier-publish_2.12/coursier-publish_2.12-2.0.0.jar
+           |@maven//:io.get-coursier/coursier-util_2.12/coursier-util_2.12-2.0.13.jar
+           |@maven//:io.get-coursier/coursier_2.12/coursier_2.12-2.0.13.jar
+           |@maven//:io.github.alexarchambault.windows-ansi/windows-ansi/windows-ansi-0.0.3.jar
+           |@maven//:io.github.alexarchambault/concurrent-reference-hash-map/concurrent-reference-hash-map-1.0.0.jar
```

Comment:
Nit: Here we could somehow be more explicit that what we were looking for was that this new transitive dependency is now included.

Reply:
Thanks for the suggestion! I added a comment that hopefully does a good job of describing what this test actually checks. You're right that it's not always obvious what these tests look for, but I'm not sure what would be the best way to improve this.

Any ideas?

Reply:
I think the comment works well. Thanks for adding it Martin!

Another idea would've been using some sort of `.includes?` assertion. That would highlight exactly what we're looking for inside the list of dependencies. But I don't think we have access to those kinds of assertions since we're testing through the checkMultipleDeps method.

I think something that makes these tests less obvious to me is that the usual test structure I'm used to (Setup Execution Assertion Teardown) is abstracted to the helper methods and thus not present when you first look at the test.

One way to improve this could be to maybe move the assertion out of the helper method and into the test so that you can, if needed, use more specific assertions that make the purpose of the test more obvious.